spec/solver_spec.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../solver'
+
+describe Solver do
+  it 'should return the factorial of N' do
+    factorial = Solver.new
+    expect(factorial.factorial(3)).to eq(6)
+    expect(factorial.factorial(0)).to eq(1)
+    expect { factorial.factorial(-1) }.to raise_error(StandardError)
+  end
+
+  it 'should return the reversed string' do
+    str_reverse = Solver.new.reverse('hello')
+    expect(str_reverse).to eql('olleh')
+  end
+
+  context 'Fizzbuzz test' do
+    fizzbuzz = Solver.new
+    it 'should return fizz if N is divisible by 3' do
+      expect(fizzbuzz.fizzbuzz(6)).to be('fizz')
+    end
+    it 'should return buzz if N is divisible by 5' do
+      expect(fizzbuzz.fizzbuzz(10)).to be('buzz')
+    end
+    it 'should return fizzbuzz if N is divisible by 3 and 5' do
+      expect(fizzbuzz.fizzbuzz(15)).to be('fizzbuzz')
+    end
```

Comment:
- [ ] These 3 tests are failing as shown in the image below 👇. A minor change such as using `expect(actual).to eq(expected)` should see your tests passing ✅ 
![image](https://user-images.githubusercontent.com/46839548/188027234-18456196-bcb9-439d-b29d-84d45daad3da.png)